iothub/service/src/Common/TrackingHelper.cs

```diff
@@ -13,10 +13,18 @@ namespace Microsoft.Azure.Devices.Common
     using Microsoft.Azure.Devices.Common.Client;
     using Microsoft.Azure.Devices.Common.Exceptions;
 
-    //TrackingId format is GUID[-G:<GatewayId>][-B:<BackendId>][-P:<PartitionId>]-TimeStamp:<Timestamp>
-
+    /// <summary>
+    /// Helper class to generate tracking Id
+    /// </summary>
+    /// <remarks>
+    /// Tracking Id format is [GUID][-G:GatewayId][-B:BackendId][-P:PartitionId][-TimeStamp:Timestamp]
+    /// </remarks>
+    [Obsolete("This is for internal use only. SDK will not support this for external usage.")]
```

Comment:
Other than 1 function, no functions in this class are being used in our SDK. This is also something that should have been internal so I've marked it as Obsolete.